x/concentrated-liquidity/lp_test.go

```diff
@@ -483,16 +482,16 @@ func (s *KeeperTestSuite) TestWithdrawPosition() {
 
 			// Create a position from the parameters in the test case.
 			if tc.createLockLocked {
-				_, _, _, liquidityCreated, _, _, err = concentratedLiquidityKeeper.CreateFullRangePositionLocked(ctx, pool.GetId(), owner, fundCoins, tc.timeElapsed)
+				_, _, _, liquidityCreated, _, _, err = concentratedLiquidityKeeper.CreateFullRangePositionLocked(s.Ctx, pool.GetId(), owner, fundCoins, tc.timeElapsed)
```

Comment:
Note: these changes were made because some helpers would update `s.Ctx` from the suite, and that change would not be propagated to the local var `ctx`, causing some test failures